runtime/bindings/python/src/pyopenvino/graph/passes/manager.cpp

```diff
@@ -39,5 +76,55 @@ void regclass_graph_passes_Manager(py::module m) {
 
     manager.def("set_per_pass_validation", &ManagerWrapper::set_per_pass_validation);
     manager.def("run_passes", &ManagerWrapper::run_passes);
-    manager.def("register_pass", (void (ManagerWrapper::*)(std::string)) & ManagerWrapper::register_pass);
+
+    // TODO: better docs
+    // manager.def(
+    //     "register_pass",
+    //     (void (ManagerWrapper::*)(std::string, py::args args, const py::kwargs& kwargs)) &
+    //         ManagerWrapper::register_pass,
+    // py::arg("pass_name"),
+    // R"(
+    //     Set the type of register pass for pass manager.
+    //     Both optional parameters are related to
+    //     'Serialize' pass.
+
+    //     Parameters
+    //     ----------
+    //     pass_name : str
+    //         String to set the type of pass
+    //     Args:
+    //         param1 : str
+    //             path where .xml file will be saved
+    //         param2 : str
+    //             path where .bin file will be saved
+
+    //     Kwargs:
+    //         xml_path : str
+    //             path where .xml file will be saved
+    //         bin_path : str
+    //             path where .bin file will be saved
```

Comment:
No need to write it down like that. Arguments should have same order when not using kwargs:)